misc/open_pdks/build-variant.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -ex
+
+VARIANT=${PKG_NAME#open_pdks.sky130}
+VARIANT=${VARIANT^^}
+./configure --prefix=$PREFIX --enable-sky130-pdk=$PREFIX/share/skywater-pdk/ --disable-alpha-sky130 --disable-xschem-sky130 --with-sky130-variants=$VARIANT
```

Comment:
Recommend wrapping this line and some comments about why you are using certain flags.

Reply:
Done